Deliver issue 2 to 4

naung9 opened the pull request:
Delivery Transporter Functions.

Commit:
Order Detail Bloc Added

Commit:
Tested With UI.

kyawswa commented:
The following need to be fixed.
- package structure
- remove unused class such as bloc_delegates.dart
- remove unnecessary comment in classes.
- user interface is not acceptable. It needs more improvement.
- The implementation does not follow the screen sketch.

I didn't find the implementation for issue #2 and #3 .

Please create pull request for each issue.

Commit:
UI improvements, Folder structure updated, components added

naung9 commented:
[x] Updated package structure
[x] Removed unused classes
[x] Removed unnecessary comments
[x] Updated UI
[x] Issue #3 Added Issue #2 Implemented as AlertDialog

Commit:
Minor Fix For Order Delivered Dialog

kyawswa commented:
Issue #2 
Please use data table for ordered Item table. I mean the UI is not looking good.
https://api.flutter.dev/flutter/material/DataTable-class.html

Order At
This date time should be **13/04/2020 13:00**

I think this is still mock data. When I accept the order and open the app again, the status won't change.
